sections/futures/Trades/Trades.tsx

```diff
@@ -1,47 +1,71 @@
-import React, { useMemo } from 'react';
-import styled, { css } from 'styled-components';
-import { CellProps } from 'react-table';
-import { Svg } from 'react-optimized-image';
-import { useTranslation } from 'react-i18next';
 import { wei } from '@synthetixio/wei';
-
+import LinkIcon from 'assets/svg/app/link.svg';
 import Card from 'components/Card';
 import Table from 'components/Table';
+import { Synths } from 'constants/currency';
+import { ETH_UNIT } from 'constants/network';
 import BlockExplorer from 'containers/BlockExplorer';
+import { format } from 'date-fns';
+import { FuturesTrade } from 'queries/futures/types';
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Svg } from 'react-optimized-image';
+import { CellProps } from 'react-table';
+import styled, { css } from 'styled-components';
 import { ExternalLink, FlexDivCentered, GridDivCenteredRow } from 'styles/common';
-
-import NoNotificationIcon from 'assets/svg/app/no-notifications.svg';
-import LinkIcon from 'assets/svg/app/link.svg';
-import { TradeStatus, PositionSide } from '../types';
-import { Synths } from 'constants/currency';
-import CurrencyIcon from 'components/Currency/CurrencyIcon';
-
-import PendingIcon from 'assets/svg/app/circle-ellipsis.svg';
-import FailureIcon from 'assets/svg/app/circle-error.svg';
-import SuccessIcon from 'assets/svg/app/circle-tick.svg';
-import { formatCurrency, formatCryptoCurrency, formatNumber } from 'utils/formatters/number';
-import { PositionHistory } from 'queries/futures/types';
+import { formatCryptoCurrency, formatCurrency } from 'utils/formatters/number';
+import { PositionSide, TradeStatus } from '../types';
 
 type TradesProps = {
-	history: PositionHistory[] | null;
+	history: FuturesTrade[] | [];
 	isLoading: boolean;
 	isLoaded: boolean;
+	marketAsset: string;
 };
 
-const Trades: React.FC<TradesProps> = ({ history, isLoading, isLoaded }) => {
+const Trades: React.FC<TradesProps> = ({ history, isLoading, isLoaded, marketAsset }) => {
 	const { t } = useTranslation();
 	const { blockExplorerInstance } = BlockExplorer.useContainer();
 
-	const columnsDeps = useMemo(() => [history], [history]);
+	const historyData = history.map((trade: FuturesTrade) => {
+		return {
+			...trade,
+			value: Number(trade?.price?.div(ETH_UNIT)),
+			amount: Number(trade?.size.div(ETH_UNIT).abs()),
+			time: Number(trade?.timestamp),
```

Comment:
Timestamp is coming through wrong due to ms:
<img width="104" alt="image" src="https://user-images.githubusercontent.com/10401554/166334852-8151663c-6527-4921-bd22-9bcedcf60b0c.png">


```suggestion
			time: Number(trade?.timestamp.div(1000),
```